src/SourceLocator/PhpInternalSourceLocator.php

```diff
@@ -43,4 +47,15 @@ private function getInternalReflectionClassName(Identifier $identifier)
 
         return $reflection->isInternal() ? $reflection->getName() : null;
     }
+
+    private function getStub($name)
+    {
+        $expectedStubName = __DIR__ . '/../../stub/' . $name . '.stub.php';
```

Comment:
In general, simply matching against `[A-Za-z]+` is enough right now, as we are dealing mostly with non-namespaced stubs only.